.github/workflows/ci.yml

```diff
@@ -14,6 +14,8 @@ jobs:
         with:
           cache: 'yarn'
       - run: yarn install --frozen-lockfile
+      # Build just the two eslint plugins needed.
+      - run: yarn nx run-many --target=build --projects='eslint-plugin-source-foundations,eslint-plugin-source-react-components'
```

Comment:
why does this need to happen in CI but not locally?

Reply:
in that other thread, when i said: 
`"and it seems that doing a build doesn't seem to be necessary for our lint process"`
I was wrong.

It seems that you need to build both of these packages as the reference is to the local versions, not those hosted via npm — meaning that the `lint` operation will only work when they have been built and are usable